builtin/common/chatcommands.lua

```diff
@@ -1,5 +1,7 @@
 -- Minetest: builtin/common/chatcommands.lua
 
+local S = core.get_translator("__builtin")
+
```

Comment:
Please add a comment here stating that `S` is <b>only</b> to be used in server code (`INIT == "game"`)